### ISSUE TITLE
[Bug]: The mobile web page cannot be opened, showing a white screen

### ISSUE THREAD

**Issue by Awhitedressy** [VueTorrent/VueTorrent#1748]
### Bug description

Mobile web page can not be opened, white screen, the computer is normal. It is now back to 2.8.1.

### Expectations

Hope to fix the problem on the mobile side

### Relevant debug informations

```shell
There is no
```

### Steps to reproduce

1. ...
2. ...
...

### How did you install VueTorrent?

Downloaded from GitHub releases

### Environment description

```plain text
Device type and name :dsm
Operating system :dsm7.2

PWA: Yes/no
Run in a container: Yes
```

### qBittorrent version

4.6.4

### VueTorrent version

2.9.0

### Agreements

- [X] The bug still occurs on the nightly build
- [X] The bug still occurs on the latest version of qBittorrent
- [X] A similar issue doesn't already exist

**Comment by Larsluph** [comment]
Without any details, we can't help you.

Did the issue also occur on Desktop ?
- If yes, please open the browser console with F12 and see if you have any errors there.
- If no, try clearing your cache / cookies / site data

**Comment by Awhitedressy** [comment]
> Without any details, we can't help you.
> 
> Did the issue also occur on Desktop ?
> 
> * If yes, please open the browser console with F12 and see if you have any errors there.
> * If no, try clearing your cache / cookies / site data

I'm running iOS 14.0. Maybe the system version is too low. The web page turns into a white screen and nothing shows when opened. There is no problem opening web pages on other phones.

**Comment by Larsluph** [comment]
Have you tried clearing your cache and site data?
Apart from that, I don't remember having introduced potential incompatible code in the codebase 🤔 I can still look at it but without the proper error it may be hard to understand and fix.

**Comment by Awhitedressy** [comment]
The cache and site data have been cleared




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Rémi ***@***.***&gt;; 
发送时间: 2024年7月19日(星期五) 晚上8:51
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [VueTorrent/VueTorrent] [Bug]: The mobile web page cannot be opened, showing a white screen (Issue #1748)














 
Have you tried clearing your cache and site data?
 Apart from that, I don't remember having introduced potential incompatible code in the codebase 🤔 I can still look at it but without the proper error it may be hard to understand and fix.

**Comment by Larsluph** [comment]
I'm working on something to allow you to create error report without the UI working, I'll let you know how it goes.

**Comment by Awhitedressy** [comment]
> I'm working on something to allow you to create error report without the UI working, I'll let you know how it goes.

Wait for update....

**Comment by Larsluph** [comment]
@Awhitedressy Here's a build including a global error handler. It should display the error in a popup when you open the page.

Linked code: #1834

[vuetorrent.zip](https://github.com/user-attachments/files/16488308/vuetorrent.zip)

**Comment by Larsluph** [comment]
Branch has been merged, this will be released in 2.12.0
